src/applications/static-pages/mhv-simple-signin-cta/index.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { signInServiceName } from '@department-of-veterans-affairs/platform-user/authentication/selectors';
+import {
+  isLoggedIn,
+  isLOA3,
+} from '@department-of-veterans-affairs/platform-user/selectors';
+import UnauthenticatedAlert from '../mhv-signin-cta/components/messages/UnauthenticatedAlert';
+import UnverifiedAlert from '../mhv-signin-cta/components/messages/UnverifiedAlert';
+
+/**
+ * MHV Signin CTA widget. This widget displays an alert if the user is not authenticated or verified.
+ * Otherwise, it displays provided HTML content.
+ * @property {HTMLElement} noAlertContent optional content to display if no alerts are shown
+ * @property {String} signInServiceName the signin service name is available
+ * @property {bool} userIsLoggedIn true if the user is logged in
+ * @property {bool} userIsVerified true if the user is verified
+ * @property {string} headingLevel the heading level
+ * @property {string} serviceDescription the heading service description
+ */
+export const MhvSimpleSigninCallToAction = ({
+  linkText,
+  linkUrl,
```

Comment:
Notice that createMhvSimpleSigninCallToAction  is calling this function without passing the `linkText` & `linkUrl`  params. Do we want to default to some initial value or alternatively, define `linkText` & `linkUrl` as required field?